src/components/Button.js

```diff
@@ -21,13 +25,32 @@ const Button = React.createClass({
 
   getDefaultProps () {
     return {
+      isActive: false,
       primaryColor: StyleConstants.Colors.PRIMARY,
       type: 'primary'
     };
   },
 
   render () {
-    const styles = {
+    const styles = this.styles();
+
+    return (
+      <div {...this.props} style={[styles.component, styles[this.props.type], this.props.style]}>
+        {this.props.icon && !this.props.isActive ? <Icon size={20} style={styles.icon} type={this.props.icon} /> : null}
```

Comment:
I think the rule is actually if there are more than three.  I think it's fine as is.

https://github.com/mxenabled/eslint-config-mx/blob/master/index.js#L158